CONTRIBUTING.md

```diff
@@ -4,6 +4,17 @@
 * Make sure you have a [GitHub account](https://github.com/signup/free)
 * Fork the repository on GitHub
 
+## Commit Guidelines
+* Separate subject from body with a blank line
+* Limit the subject line to 50 characters- it's a rule of thumb. Follow it strictly
+  as long as possible, otherwise long subject line will get truncated.
+* Capitalize the subject line
+* Do not end the subject line with a period
+* Use the imperative mood in the subject line i.e. as you are giving order.
+* keep the length of line in body at 72 character.
+* Use the body to explain what and why vs how in consize manner i.e. no story telling 
+  and no inferential statements. Explain these three things to the point.
```

Comment:
"Explain these three things to the point" only reiterates the first part of this bullet point. Remove it.